src/redux/store.js

```diff
@@ -0,0 +1,13 @@
+import { configureStore } from "@reduxjs/toolkit";
+import BurgerBuilderSlice from "./slices/burgerBuilderSlice";
+import authSlice from "./slices/authSlice";
+import orderDeliveryDetalis from "./slices/orderDerliveryDetails";
+import orderSlice from "./slices/orderSlice";
+export const store = configureStore({
+  reducer: {
+    BurgerBuilderSlice: BurgerBuilderSlice,
```

Comment:
it should be `burgerBuilderSlice`, If a property start with capital letter, React thinks it is a JSX element that's why it is showing as orange. make sure to follow proper naming conventions as much as possible.